diff -ru xtrkcad-source-5.2.2GA-orig/app/lib/CMakeLists.txt xtrkcad-source-5.2.2GA/app/lib/CMakeLists.txt
--- xtrkcad-source-5.2.2GA-orig/app/lib/CMakeLists.txt	2022-01-13 14:24:39.874661716 -0500
+++ xtrkcad-source-5.2.2GA/app/lib/CMakeLists.txt	2022-01-13 10:12:54.291054979 -0500
@@ -15,6 +15,7 @@
 	logo.bmp
 	xtrkcad.xtq
 	xtrkcad.upd
+	xtrkcad.desktop
 	DESTINATION ${XTRKCAD_SHARE_INSTALL_DIR}
 	)
 	
